maps/src/test/java/org/odk/collect/maps/LineDescriptionTest.kt

```diff
@@ -0,0 +1,58 @@
+package org.odk.collect.maps
+
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import org.hamcrest.CoreMatchers.equalTo
+import org.hamcrest.MatcherAssert.assertThat
+import org.junit.Test
+import org.junit.runner.RunWith
+
+@RunWith(AndroidJUnit4::class)
+class LineDescriptionTest {
+    @Test
+    fun `getStrokeWidth returns the default value when the passed one is null`() {
+        val lineDescription = LineDescription(emptyList(), null, null, false, false)
+        assertThat(lineDescription.getStrokeWidth(), equalTo(MapConsts.DEFAULT_STROKE_WIDTH))
+    }
+
+    @Test
+    fun `getStrokeWidth returns the default value when the passed one is invalid`() {
+        val lineDescription = LineDescription(emptyList(), "blah", null, false, false)
+        assertThat(lineDescription.getStrokeWidth(), equalTo(MapConsts.DEFAULT_STROKE_WIDTH))
+    }
+
+    @Test
+    fun `getStrokeWidth returns the default value when the passed one is not greater than or equal to zero`() {
+        val lineDescription = LineDescription(emptyList(), "-1", null, false, false)
+        assertThat(lineDescription.getStrokeWidth(), equalTo(MapConsts.DEFAULT_STROKE_WIDTH))
+    }
+
+    @Test
+    fun `getStrokeWidth returns custom value when the passed one is a valid int number`() {
+        val lineDescription = LineDescription(emptyList(), "10", null, false, false)
+        assertThat(lineDescription.getStrokeWidth(), equalTo(10f))
+    }
+
+    @Test
+    fun `getStrokeWidth returns custom value when the passed one is a valid float number`() {
+        val lineDescription = LineDescription(emptyList(), "10.5", null, false, false)
+        assertThat(lineDescription.getStrokeWidth(), equalTo(10.5f))
+    }
+
+    @Test
+    fun `getStrokeColor returns the default color when the passed one is null`() {
+        val lineDescription = LineDescription(emptyList(), null, null, false, false)
+        assertThat(lineDescription.getStrokeColor(), equalTo(-65536))
```

Comment:
`-65536` should be a constant (which also helps places where we translate the string color).

Reply:
Done.